### PR TITLE
[MBL-878] Use compose to show ChangePasswordActivity

### PR DESCRIPTION
**Pull request by mtgriego**
# 📲 What

Create compose visuals and use them for ChangePasswordActivity

# 🤔 Why

Compose everything everywhere

# 🛠 How

Using designer components and some custom work

# 👀 See

Screenshot_20230717-154738
Screenshot_20230717-154726
Screenshot_20230717-154658
Screenshot_20230717-154645

# 📋 QA

Go to the change password screen and confirm it behaves 1:1 with the preview visual version

# Story 📖

[MBL-878](https://kickstarter.atlassian.net/browse/MBL-878)

[MBL-878]: https://kickstarter.atlassian.net/browse/MBL-878?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by Arkariang — APPROVED** [review]
Amazing job!

File: app/src/main/java/com/kickstarter/ui/activities/ChangePasswordActivity.kt [hunk 1]
```diff
@@ -40,71 +40,50 @@ class ChangePasswordActivity : AppCompatActivity() {
...
-            .addToDisposable(disposables)
 
-        this.viewModel.outputs.passwordWarning()
-            .compose(Transformers.observeForUIV2())
-            .subscribe {
-                binding.warning.text = when {
-                    it == 0 -> null
-                    it != null -> getString(it)
-                    else -> null
+            this.viewModel.outputs.progressBarIsVisible()
```
Comment (Arkariang) [inline app/src/main/java/com/kickstarter/ui/activities/ChangePasswordActivity.kt:63]: fyi, if you like you could use `subscribeAsState` and work with State, you have an example on `ReportProjectActivity:82`

Screenshot 2023-07-18 at 7 57 17 AM

https://developer.android.com/jetpack/compose/state#use-other-types-of-state-in-jetpack-compose

File: app/src/main/java/com/kickstarter/ui/activities/ChangePasswordActivity.kt [hunk 2]
```diff
@@ -40,71 +40,50 @@ class ChangePasswordActivity : AppCompatActivity() {
...
-                binding.progressBar.isGone = !it
+        setContent {
+            // Replace with feature flag
+            var darkMode by remember { mutableStateOf(false) }
+
+            var showProgressBar by remember { mutableStateOf(false) }
+
+            var errorMessage by remember { mutableStateOf("") }
+
+            KSTheme(useDarkTheme = darkMode) {
```
Comment (Arkariang) [inline app/src/main/java/com/kickstarter/ui/activities/ChangePasswordActivity.kt:51]: 👍

File: app/src/main/java/com/kickstarter/ui/activities/ChangePasswordActivity.kt (hunk 1 above)
Comment (mtgriego) [reply app/src/main/java/com/kickstarter/ui/activities/ChangePasswordActivity.kt:63]: oooo, i didnt know about this, will look into it!

**Pull request merged**